pkg/chains/formats/intotoite6/intotoite6.go

```diff
@@ -58,260 +44,16 @@ func (i *InTotoIte6) Wrap() bool {
 }
 
 func (i *InTotoIte6) CreatePayload(obj interface{}) (interface{}, error) {
-	var tr *v1beta1.TaskRun
 	switch v := obj.(type) {
-	case *v1beta1.TaskRun:
-		tr = v
+	case *objects.TaskRunObject:
+		return taskrun.GenerateAttestation(i.builderID, v, i.logger)
```

Comment:
Double check this, but I don't think logger is used? (it might be generally better to just return an error and log it here anyway)

Reply:
Looks like it's used in [getting the subject digests](https://github.com/hacbs-contract/chains/blob/poc-tep-84/pkg/chains/formats/intotoite6/util/utils.go#L63), logging an error that it's not in the correct format.